v3/lints/cabf_ev/lint_ev_organization_id_missing_test.go

```diff
@@ -0,0 +1,39 @@
+package cabf_ev
+
+/*
+ * ZLint Copyright 2020 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+var tests = map[string]lint.LintStatus{
```

Comment:
tiny nit: is there a reason to make this a global instead of a var scoped to `TestOrganizationIDMissing`?

Reply:
None whatsoever, making in locally scoped now.